example/App.re

```diff
@@ -23,7 +23,8 @@ module Values = {
 
 // It's pretty easy to have a generic form maker
 // for when a the same value set and errors are used in different context
-let makeForm = Formidable.make((module Values), (module I18n.Error));
+let makeForm =
+  Formidable.make(~values=(module Values), ~error=(module I18n.Error));
```

Comment:
actually prefer this one :D

Reply:
Definitely clearer this way ^^
Notice that the constructor in dr-front uses labels :smiley: